### DISCUSSION TITLE
Question about rounding time in executiontime block

### DISCUSSION THREAD

**Discussion by tleepa** [JanDeDobbeleer/oh-my-posh#722] [General]
Is there an option/workoaround to round the execution time to seconds?
E.g. `3s` instead of `3.011s` :)

**Comment by JanDeDobbeleer** [comment]
Coming up in #727

**Reply by tleepa** [reply]
That was quick! :)

If I read the code correctly, the new round format shows only one number - seconds/minutes/hours/etc.
So, execution time of 3 hours and 5 minutes will show `3h`.
But execution time of 3 hours and 55 minutes will also shoe `3h` - right? Will show `4h` only when execution time is greater than 4 hours.

I was basically looking for austin format without fractional part, e.g. `5s`, `3m 1s`, `4h 35m 55s`, etc. - sorry for not being precise enough with my original question.

Also I am not saying that the new `round` format is bad :)

**Reply by JanDeDobbeleer** [reply]
That can be done too. It's all a matter of preference 😬

**Reply by JanDeDobbeleer** [reply]
I made a fix in #728 , the only thing that didn't make sense to me directly was `4h 35m 55s` which is now `4h 35m` as the seconds don't really matter anymore in that context.

**Reply by tleepa** [reply]
Yes, you are probably right. I am trying to get the same look as I have on ohmyzsh's powerlevel10k theme and I cannot recall  seeing times longer than 1 hour, so I am not sure how it looks like there.
But your change makes a lot of sense.

Thank you for your effort :)
